### ISSUE TITLE
Scrape config for blackbox dex uses the wrong healthz endpoint

### ISSUE THREAD

**Issue by SayakMukhopadhyay** [gardener/garden-setup#231]
**What happened**:
Blackbox is using the wrong endpoint for healthz for dex. Thus grafana is showing dex to be down.
**What you expected to happen**:
Grafana showing dex to be up since its working
**How to reproduce it (as minimally and precisely as possible)**:
Deploy the current setup and check the status of dex in garden grafana
**Anything else we need to know?**:
The issue is in this line https://github.com/gardener/garden-setup/blob/53df009b155edd13cd09ee258aa2b32a89a77449/components/monitoring/prometheus/manifests/scrape-configs/scrape-blackbox-dex.yaml#L11

`identity_url` needs to be replaced with `issuer_url`
**Environment**:

- Version of garden-setup (release or commit): 1.32
- Versions of components (only needed if you overwrote the defaults)
- Where does the underlying base cluster come from and which operating system does it use? AKS-Ubuntu 18.04
- Which cloud provider is configured for the setup? Azure
- Other:

**Comment by SayakMukhopadhyay** [comment]
I am applying the change in my cluster and once I verify its working, I will submit a PR

**Comment by Diaphteiros** [comment]
Hm, I tried to reproduce this, but my Grafana shows dex/identity as being up. Did you change the issuerUrl in your acre.yaml? Maybe it works for the default ... ?

**Comment by SayakMukhopadhyay** [comment]
Yeah, I did change the `issuerUrl` to `https://identity.ing.<landscapedomain>/oidc`. I was seeing that the dashboard was trying to get results from `https://identity.ing.<landscapedomain>/healthz` which would give an ingress error. Let me try to set up a cluster with no issuerUrl and I will get back to you.

**Comment by SayakMukhopadhyay** [comment]
I checked, and it seems like it is working _somehow_. The Grafana dashboard is fetching from the `identity.ing...` url although that URL is not supposed to work since the identity is at `gardener.ing.../oidc`. Yet this URL sends a `200 OK` with a 0 length response.

This is the Grafana request's response

```
{
    "status": "success",
    "data": {
        "resultType": "matrix",
        "result": [
            {
                "metric": {
                    "__name__": "probe_success",
                    "instance": "https://identity.ing.<domain>/healthz",
                    "job": "blackbox-dex",
                    "purpose": "availability"
                },
                "values": [
                    [
                        1590659040,
                        "1"
                    ],
```
And this is the result of `wget` on the above `instance` url

```
Resolving identity.ing.<domain> (identity.ing.<domain>)... 52.224.33.51
Connecting to identity.ing.<domain> (identity.ing.<domain>)|52.224.33.51|:443... connected.
WARNING: cannot verify identity.ing.<domain>'s certificate, issued by ‘CN=Kubernetes Ingress Controller Fake Certificate,O=Acme Co’:
  Unable to locally verify the issuer's authority.
WARNING: no certificate subject alternative name matches
        requested host name ‘identity.ing.<domain>’.
HTTP request sent, awaiting response... 200 OK
Length: 0 [text/html]
Saving to: ‘healthz’

healthz                                   [ <=>                                                                     ]       0  --.-KB/s    in 0s

2020-05-28 10:00:14 (0.00 B/s) - ‘healthz’ saved [0/0]

```
But if I make a query at `gardener.ing.<domain>/oidc/healthz` I am getting a response body `Health check passed in 23.081096ms` which indicates that this is the correct URL which in turn indicates that the `issuer_url` should be the correct paramater.

**Comment by Diaphteiros** [comment]
Fixed by #232
